packages/cli/__tests__/auth/__system__/cli.auth.login.apiml.system.test.ts

```diff
@@ -104,23 +102,42 @@ describe("auth login/logout apiml show token", () => {
         expect(response.status).toBe(0);
         const responseData = JSON.parse(response.stdout.toString()).data;
         expect(responseData.tokenType).toEqual("apimlAuthenticationToken");
-        expect(responseData.tokenValue).not.toEqual(token[token.length-3]); // Intermittent failure : (
-        expect(responseData.tokenValue.length).toEqual(token[token.length-3].length);
+        expect(responseData.tokenValue).toBeDefined();
+        expect(responseData.tokenValue.length).toBeGreaterThan(100);
```

Comment:
If the intent of this test is to validate the token returned from the API ML, should we call `ConfigUtils.hasTokenExpired` to check that we have a non-expired JWT?

Reply:
I am not sure why the original test implementation tested for length, but I can replace it with hasTokenExpired. That seems to better reflect the intent.